jetty-osgi/jetty-osgi-boot/src/main/java/org/eclipse/jetty/osgi/boot/internal/serverfactory/DefaultJettyAtJettyHomeHelper.java

```diff
@@ -174,14 +175,34 @@ else if (jettyHomeBundleSysProp != null)
             }
             Thread.currentThread().setContextClassLoader(cl);
 
-            // these properties usually are the ones passed to this type of
-            // configuration.
+            //the default server name
             properties.put(OSGiServerConstants.MANAGED_JETTY_SERVER_NAME, OSGiServerConstants.MANAGED_JETTY_SERVER_DEFAULT_NAME);
-            Util.setProperty(properties, OSGiServerConstants.JETTY_HOST, System.getProperty(OSGiServerConstants.JETTY_HOST, System.getProperty("jetty.host")));
-            Util.setProperty(properties, OSGiServerConstants.JETTY_PORT, System.getProperty(OSGiServerConstants.JETTY_PORT, System.getProperty("jetty.port")));
-            Util.setProperty(properties, OSGiServerConstants.JETTY_PORT_SSL, System.getProperty(OSGiServerConstants.JETTY_PORT_SSL, System.getProperty("ssl.port")));
+            
+            //Always set home and base
             Util.setProperty(properties, OSGiServerConstants.JETTY_HOME, home);
             Util.setProperty(properties, OSGiServerConstants.JETTY_BASE, base);
+            
+            // copy all system properties starting with "jetty." to service properties for the jetty server service.
+            // these will be used as xml configuration properties.
+            for (Map.Entry<Object, Object> prop : System.getProperties().entrySet())
+            {
+                if (prop.getKey() instanceof String)
+                {
+                    String skey = (String)prop.getKey();
+                    //never copy the jetty xml config files into the properties as we pass them explicitly into
+                    //the call to configure
+                    if (OSGiServerConstants.MANAGED_JETTY_XML_CONFIG_URLS.equals(skey))
```

Comment:
I think we should also skip `OSGiServerConstants.JETTY_HOME` and `OSGiServerConstants.JETTY_BASE` as we are setting these explicitly above and don't want them overwritten if they are a system property.